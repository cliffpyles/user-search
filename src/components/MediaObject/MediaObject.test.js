import React from 'react'
import Renderer from 'react-test-renderer'
import MediaObject from './'

test('can render a MediaObject', () => {
  const component = Renderer.create(<MediaObject />)

  expect(component.toJSON()).toMatchSnapshot()
})
