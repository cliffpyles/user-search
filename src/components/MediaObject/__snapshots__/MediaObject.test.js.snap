// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render a MediaObject 1`] = `
<div
  className="MediaObject"
>
  <div
    className="MediaObjectMedia"
  />
  <div
    className="MediaObjectContent"
  />
</div>
`;
